spec/system/create_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'Kudo management', type: :system do
```

Comment:
What do you think about renaming this file to `kudo_create_spec.rb`?
Later when there will be more objects to create it can be confusing looking at this filename and trying to figure out what this spec is creating